backend/src/controllers/fortuneTeller/fortuneTeller.controller.ts

```diff
@@ -1,5 +1,60 @@
 import { Request, Response } from "express"
 import { fortuneTellerService } from "../../services/fortuneTeller/fortuneTeller.services"
+import { FortuneTellerRegisterSchema, RequestSchema } from "../../models/fortuneTeller/fortuneTeller.model"
+
+const createFortuneTellerRegister = async (req: Request, res: Response) => {
+  const fortuneTeller: FortuneTellerRegisterSchema = {
+    fortunetellerid: req.body.fortunetellerid,
+    identitycardnumber: req.body.identitycardnumber,
+    identitycardcopy: req.body.identitycardcopy
+  }
+
+  const result = await fortuneTellerService.createFortuneTellerRegister(fortuneTeller)
+
+  if (!result.success) return res.status(400).json(result)
+  res.status(201).json({ success: true })
+}
+
+const createFortuneTellerRequest = async (req: Request, res: Response) => {
+  const request: RequestSchema = {
+    fortune_teller_id: req.body.fortunetellerid,
+    status: "PENDING"
+  }
+
+  const result = await fortuneTellerService.createRequest(request)
+
+  if (!result.success) return res.status(400).json(result)
+  res.status(201).json(result)
+}
+
+const updateFortuneTellerRegister = async (req: Request, res: Response) => {
+  const fortuneTeller: FortuneTellerRegisterSchema = {
+    fortunetellerid: req.body.fortunetellerid,
+    identitycardnumber: req.body.identitycardnumber,
+    identitycardcopy: req.body.identitycardcopy
+  }
+  const resultOne = await fortuneTellerService.updateFortuneTellerRegister(fortuneTeller)
+
+  if (!resultOne.success) return res.status(400).json(resultOne)
+
+  const request: RequestSchema = {
+    fortune_teller_id: req.body.fortunetellerid,
+    status: "PENDING"
+  }
+
+  const resultTwo = await fortuneTellerService.updateFortuneTellerRequest(request)
+
+  if (!resultTwo.success) return res.status(400).json(resultTwo)
+  res.status(200).json({ success: true })
+}
+
+const getFortuneTellerValid = async (req: Request, res: Response) => {
+  const fortunetellerid: string = req.body.fortunetellerid
+
+  const result = await fortuneTellerService.getFortuneTellerValid(fortunetellerid)
+
+  return res.status(200).json(result)
```

Comment:
result is not a json please fix to { success: boolean, data?: SomeType, message?: string}